### DISCUSSION TITLE
Led dimmer and contact sensor

### DISCUSSION THREAD

**Discussion by lukaszwitkowski** [RavenSystem/esp-homekit-devices#1657] [MEPLHAA Scripts]
Hello i need to help with my config
I Want led dimmer , button , and contact sensor (set state when door open)
I have problem becouse my contact senson send "open" status for only 3 second i want to have open status until contact senson (door is open)

Help

{
  "c":{ "l": 13,"q":500,"b":[{"g":0,"t":5}]},
  "a":[{
    "t":30,
    "g":[15],
    "b":[{"g":0}]
  },
{
    "t": 5,
    "i": 3,
    "f1": [{"g":2}]
  }]
}

**Comment by RavenSystem** [comment]
{
"c":{ "l": 13,"q":500,"b":[{"g":0,"t":5}]},
"a":[{
"t":30,
"g":[15],
"b":[{"g":0}]
},
{
"t": 5,
"f0": [{"g":2,"t":0}],
"f1": [{"g":2}]
}]
}

**Comment by lukaszwitkowski** [comment]
Wow works great:) 
Thanks
